etc/bento_deploy.env

```diff
@@ -0,0 +1,39 @@
+BENTOV2_DOMAIN=bentov2.local
+BENTOV2_PORTAL_DOMAIN=portal.${BENTOV2_DOMAIN}
+BENTOV2_AUTH_DOMAIN=bentov2auth.local
+
+MODE=dev
+DOLLAR=$
+
+BENTOV2_USE_EXTERNAL_IDP=0
+BENTOV2_USE_BENTO_PUBLIC=1
+BENTOV2_PRIVATE_MODE=false
+
+BENTOV2_ROOT_DATA_DIR=~/bentov2/data
+
+BENTOV2_AUTH_ADMIN_USER=
+BENTOV2_AUTH_ADMIN_PASSWORD=
+
+BENTOV2_AUTH_TEST_USER=
+BENTOV2_AUTH_TEST_PASSWORD=
+
+CLIENT_SECRET=
+
+BENTO_WEB_CUSTOM_HEADER=
+
+BENTOV2_KATSU_DB_USER=admin
+BENTOV2_KATSU_DB_PASSWORD=devpassword123
+
+BENTOV2_WEB_TAG_OR_BRANCH=v1.3.0
+BENTOV2_DROP_BOX_TAG_OR_BRANCH=v0.5.0
+BENTOV2_SERVICE_REGISTRY_TAG_OR_BRANCH=v0.5.1
+BENTOV2_KATSU_TAG_OR_BRANCH=v2.12.0
+BENTOV2_LOGGING_TAG_OR_BRANCH=v0.1.1
+BENTOV2_DRS_TAG_OR_BRANCH=v0.5.3
+BENTOV2_NOTIFICATION_TAG_OR_BRANCH=v1.0.0
+BENTOV2_FEDERATION_TAG_OR_BRANCH=v0.13.1
+BENTOV2_EVENT_RELAY_TAG_OR_BRANCH=v1.0.0
+BENTOV2_WES_TAG_OR_BRANCH=v0.5.1
+
+GOHAN_TAG_OR_BRANCH=v3.5.1
+BENTO_PUBLIC_TAG_OR_BRANCH=v0.5.0
```

Comment:
These must be commented out. In the ideal case, one should never edit those and get the updated versions only from the git managed env file